static/app/utils/useDevicePixelRatio.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+/**
+ * Hook to retrieve dpr value of the device and monitor for changes
+ * (e.g. if user drags window to a screen with different dpr, we want to be notified).
+ * @returns dpr of the device
+ */
+function useDevicePixelRatio(): number {
+  const [devicePixelRatio, setDevicePixelRatio] = React.useState<number>(
+    window.devicePixelRatio
+  );
+
+  const updatePixelRatio = React.useCallback(() => {
+    setDevicePixelRatio(window.devicePixelRatio);
+  }, []);
```

Comment:
Missing `setDevicePixelRatio` in dependencies.

Reply:
It's stable, so not necessary. I think we should keep these out of the dependencies so we dont have too much noise there